server/service/IndividualServicesService.js

```diff
@@ -321,27 +323,38 @@ exports.regardApplication = function (body, user, originator, xCorrelator, trace
        * Setting up required local variables from the request body
        ****************************************************************************************/
       let applicationName = body["application-name"];
-      let releaseNumber = body["application-release-number"];
-      let applicationAddress = body["application-address"];
-      let applicationPort = body["application-port"];
+      let releaseNumber = body["release-number"];
+      let tcpServerList = [
+        {
+          protocol : body["protocol"],
+          address : body["address"],
+          port : body["port"]
+      }
+      ];
       let inquireOamRequestOperation = "/v1/inquire-oam-request-approvals";
+      let operationNamesByAttributes = new Map();
+      operationNamesByAttributes.set("inquire-oam-request-approvals", inquireOamRequestOperation);
 
+      let tcpObjectList = [];
```

Comment:
Assumption is it will work without formulateTcpObject method and adding it to the list again. Kindly ,check without the method.